zone/client.cpp

```diff
@@ -13597,3 +13597,18 @@ void Client::CheckItemDiscoverability(uint32 item_id)
 
 	DiscoverItem(item_id);
 }
+
+void Client::SetIllusionBlock(bool value, bool save) {
```

Comment:
I'm not really a fan of the optional bool `save` pattern. A function should generally have a single responsibility and one reason to change. ([SRP](https://en.wikipedia.org/wiki/Single-responsibility_principle))

Characters will hit Save() periodically and also when when they zone/camp/save so this will persist anyways. Let's strip this to simplify

Reply:
Similarly, this field needs to be added in `ZoneDatabase::SaveCharacterData`